resources/definitions/robo_3d_r1.def.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "robo_3d_r1",
+    "name": "Robo R1",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Ultimaker",
+        "manufacturer": "Robo 3D",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "platform_offset": [ 0, 0, 0]
+    },
+
+    "overrides": {
+        "machine_start_gcode": {
+            "default_value": " G92 E0 ;\n M565 Z-1 ;\n G1 Z5 F5000 ;\n G29 ;\n"
+        },
+        "machine_end_gcode": {
+            "default_value": " M104 S0                     ;extruder heater off\n M140 S0                     ;heated bed heater off (if you have it)\n G91                                    ;relative positioning\n G1 E-1 F300                            ;retract the filament a bit before lifting the nozzle, to release some of the pressure\n G1 Z+0.5 E-5 X-20 Y-20 F{travel_speed} ;move Z up a bit and retract filament even more\n G28 X0 Y0                              ;move X/Y to min endstops, so the head is out of the way\n M84                         ;steppers off\n G90                         ;absolute positioning\n ;{profile_string}\n"
+        },
+        "cool_min_layer_time": { "default_value": 7 },
+        "speed_topbottom": { "default_value": 40 },
+        "retraction_speed": { "default_value": 50 },
+        "layer_0_z_overlap": { "default_value": 0.2 },
+        "cool_min_speed": { "default_value": 19 },
+        "material_bed_temperature": { "default_value": 60 },
+        "support_angle": { "default_value": 50 },
+        "speed_layer_0": { "default_value": 30 },
+        "line_width": { "default_value": 0.39999999999999997 },
```

Comment:
I don't think those 0.03am are going to make any difference! 0.4 is the default.